pegasus/table_connector.go

```diff
@@ -775,6 +778,33 @@ func (p *pegasusTableConnector) SortKeyCount(ctx context.Context, hashKey []byte
 	return count, WrapError(err, OpSortKeyCount)
 }
 
+func (p *pegasusTableConnector) Incr(ctx context.Context, hashKey []byte, sortKey []byte, increment int64) (int64, error) {
+	newValue, err := func() (int64, error) {
+		if err := validateHashKey(hashKey); err != nil {
+			return 0, err
+		}
+		if err := validateSortKey(sortKey); err != nil {
+			return 0, err
+		}
+
+		gpid, part := p.getPartition(hashKey)
+
+		request := rrdb.NewIncrRequest()
+		request.Key = encodeHashKeySortKey(hashKey, sortKey)
+		request.Increment = increment
```

Comment:
Please add `request.ExpireTsSeconds = 0` here, for clarity.

Reply:
> Please add `request.ExpireTsSeconds = 0` here, for clarity.

It's seem no such field named `ExpireTsSeconds` in `rrdb.IncrRequest`.

Reply:
fine, I will fix it later.